LogseqPDFImporter.py

```diff
@@ -287,7 +287,11 @@ def main(
                     words,
                     keep_newlines,
                     text_boundary_threshold)
-            annotdict["contents"] = text
+            comment_text = annot.info["content"]
```

Comment:
Please add a check so that we don't crash if there is no .info and use .get("content", "").